rrat/rrat/urls.py

```diff
@@ -24,5 +24,8 @@
     path('about/', views.about, name='about'),  # About Page
     path('patients/', include(('patients.urls', 'patients'), namespace='patients')),
     path('retina_photos/', include(('retina_photos.urls', 'retina_photos'), namespace='retina_photos')),
+    path('', views.homepage, name='home'),  # Homepage
+    path('about/', views.about, name='about'),  # About Page
+    path('patients/', include('patients.urls')),
```

Comment:
looks like you had a merge conflict and left duplicate entries. We should probably remove these.